src/app/js/views/table.js

```diff
@@ -24,3 +17,11 @@ default class TableView extends Component {
     )
   }
 }
+
+Table.displayName = 'Table'
```

Comment:
If I understand it correctly, the only reason why the previous version was working (https://github.com/ipfs/webui/pull/595#discussion_r123906426 and https://github.com/ipfs/webui/pull/595#discussion_r123906457) is thanks to some transpilation step as JS doesn't naturally support it

![image](https://user-images.githubusercontent.com/1211152/27519499-09a7484a-59ed-11e7-82ad-2c1a00ee26d5.png)